tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py35-{syntax,unit-tests,build-binary}
+envlist = {py35|py36}-{syntax,unit-tests,build-binary}
```

Comment:
The test results show that this was treated as a single interpreter, not a command to use both interpreters. Looks like it should be a comma: https://tox.readthedocs.io/en/latest/example/basic.html#compressing-dependency-matrix

Reply:
I had originally tried using `,` but it looks comma-delimiters will attempt a build on each interpreter (in this case, `py35` and `py36`).  Our TeamCity agent performing builds does not have 3.6 installed, which causes `tox` to [fail](https://teamcity.mesosphere.io/viewLog.html?buildId=814035&buildTypeId=DcosIo_DcosLaunch_Build&tab=buildLog&_focus=604).  Using `|` as the delimiter will build based on what is available.